posts/2021-01-14-free-nodejs-hosting.md

```diff
@@ -98,87 +102,72 @@ It also has comprehensive [docs](https://render.com/docs?utm_source=geshan.com.n
+<img class="center" loading="lazy" src="/images/free-nodejs-hosting/05genez-create-repo.jpg" title="Create repo in genezio" alt="Create repo in genezio">
 
-* It will build and deploy the app and you will see a screen like below:
+- Subsequently, Genezio will build and deploy the app for you in a live URL in a couple of minutes.
 
-<img class="center" loading="lazy" src="/images/free-nodejs-hosting/09nodejs-vercel-deployed.jpg" title="The app is deployed on Vercel" alt="The app is deployed on Vercel">
+<img class="center" loading="lazy" src="/images/free-nodejs-hosting/05genez-live-url.jpg" title="Genezio URL" alt="Genzio URL">
 
-* Now, click the “Visit” button, you should see something like the following:
+- To view the working app click on the URL on overview page
 
-<img class="center" loading="lazy" src="/images/free-nodejs-hosting/10nodejs-vercel-app-running.jpg" title="The app is running on Vercel" alt="The app is running on Vercel">
+<img class="center" loading="lazy" src="/images/free-nodejs-hosting/05genez-overview.jpg" title="Genezio Dashboard" alt="Genzio dashboard">
 
-* Add `/quotes` to the URL and you should see the quotes in a JSON format like below:
+- It should come up something like this
 
-<img class="center" loading="lazy" src="/images/free-nodejs-hosting/11nodejs-vercel-api-running.jpg" title="The quotes API is running on Vercel" alt="The quotes API is running on Vercel">
+<img class="center" loading="lazy" src="/images/free-nodejs-hosting/05genez-live-website.jpg" title="Live website deployed in Genezio" alt="Live website deployed in Genezio">
 
-This is set up very well now, you should go to the app’s settings page if you want to tweak anything. You can add environment variables and do other things in the settings. You can even look at the function’s logs.
+- You can also edit your code directly in Genezio or get help on your code using Genezio AI built in.
 
-Each time you open a new pull request in that repository Vercel will give you a deploy preview URL which is very handy. You can see how it looks like below:
-
-<img class="center" loading="lazy" src="/images/free-nodejs-hosting/12nodejs-vercel-deploy-preview.jpg" title="Vercel deploy preview on GitHub pull request" alt="Vercel deploy preview on GitHub pull request">
-
-On merge to master/main branch, it will auto-deploy the code to the main URL. You can use the [Vercel CLI](https://vercel.com/docs/cli) for re-deployments, setting up environment variables, and a host of other things.
-
-Vercel has a powerful CLI to make your life easier for deploying the apps and doing other things like having a look at the app logs from the CLI.
-
-Next, let's look at how Cyclic compares to Render and Vercel.
+<img class="center" loading="lazy" src="/images/free-nodejs-hosting/05genez-ai.jpg" title="Chat with Genezio AI" alt="Chat with Genezio Ai">
```

Comment:
Same here only the right part seems interesting and can be taken at 800 px browser not your full screen browser.